packages/core/dispatcher/dispatcher.js

```diff
@@ -36,7 +38,12 @@ const setDefaultTargetById = (id) => {
  * @memberof core/dispatcher
  * @private
  */
-const getDefaultTarget = () => defaultTarget;
+const getDefaultTarget = () => {
+	if (!defaultTarget) {
+		setDefaultTargetById(rootId);
```

Comment:
Should we add some documentation for this behavior to the above or not?

Reply:
I fixed